### DISCUSSION TITLE
Installing spacy transformers via conda/mamba

### DISCUSSION THREAD

**Discussion by JovanVeljanoski** [explosion/spaCy#10411] [Help: Installation]
Hi all, 

I would like to ask if it is possible or if there are any plans to make spacy-transformers available on conda-forge, so that it can be installed via conda/mamba? 

Technically, it is there, but (for me)  the installation fails because a dependency (spacy-alignments) is not available on conda-forge. 

Thank you!

OS: Mac OS (M1)
Python: 3.9
Env: conda/mamba.

**Comment by adrianeboyd** [comment]
We have added a `spacy-transformers` package to `conda-forge` relatively recently. Usually the additional architectures are added automatically to the builds not too long after the packages are first published, but sometimes a bot error or a missing dependency holds things up. I checked the `conda-forge` status and couldn't immediately figure out why `spacy-alignments` hasn't been processed yet for osx arm64.

A lot of the details are out of our hands here, but I'll see if I can find out a bit more about what's going on...

**Reply by JovanVeljanoski** [reply]
Thank you - appreciate it. I know that `conda-forge` recipes can be tricky at times..
